src/server/utilities/cspHeader/index.js

```diff
@@ -12,64 +12,41 @@ import { bbcDomains, advertisingServiceCountryDomains } from './domainLists';
 const advertisingDirectives = {
   connectSrc: [
     'https://csi.gstatic.com',
-    'https://experience.tinypass.com',
-    'https://pagead2.googlesyndication.com',
-    'https://*.g.doubleclick.net',
-    'https://survey.effectivemeasure.net',
-    'https://collector.effectivemeasure.net',
-    'https://detect-survey.effectivemeasure.net',
+    'https://*.googlesyndication.com',
+    'https://*.doubleclick.net',
+    'https://*.effectivemeasure.net',
```

Comment:
Looks like we've got quite a few wildcard csp entries being introduced here, is this 'tidying up' or strictly necessary, it would better to keep the specific entries as they were if the currently serve the required purpose.

Reply:
I thought it'd simplify things, and make the csp file smaller... I can go back to fix entry...

Reply:
It technically make the csp less secure as it allows more domains, as a guideline wildcard usage in csps should be minimized. I agree it's more readable and tidy but technically less secure unfortunately